CHANGELOG.md

```diff
@@ -27,6 +28,34 @@
  - [0.0.5](#005---20180602)
  - [0.0.4 and prior](#004-and-prior)
 
+## [2.0.0] - WIP
```

Comment:
Now we know that we'll mark the new release as 2.0.0 when it's done, but maybe consistently using "next release" would be more convenient and allow us to decide on the version number as we cut the release?

Just a thing to consider.

Reply:
```suggestion
## [Next Release]
```